### PR TITLE
Remove string.quote. Fixes: #1576

### PR DESCRIPTION
**Pull request by toots**

### CODE REVIEW

**Comment by smimram** [comment]
Sorry to be picky, but I would rather write `string.quote` than `process.quote`. Would it be possible to change both to the current behavior of `process.quote` and deprecate `process.quote` instead?

**Comment by toots** [comment]
Done!

File: src/lang/builtins_string.ml [hunk 1]
```diff
@@ -394,10 +394,14 @@ let () =
       Lang.string (Utils.interpolate (fun k -> List.assoc k l) s))
 
 let () =
-  add_builtin "string.quote" ~cat:String ~descr:"Escape shell metacharacters."
+  add_builtin "string.quote" ~cat:String
+    ~descr:
+      "Return a quoted copy of the given string, suitable for use as one \
```
Comment (smimram) [inline src/lang/builtins_string.ml:399]:
```suggestion
      "Return a quoted copy of the given string, suitable for use as an \
```

**Review by smimram — APPROVED** [review]
Go!

**Pull request merged**